.github/workflows/docker.yml

```diff
@@ -46,8 +46,11 @@ jobs:
       matrix:
         include:
           - build_args: ""
+            image_tag: "latest"
+          - build_args: "--build-arg REFERENCE_DEVICE=1 --build-arg BACKBONE_ROUTER=1 --build-arg NAT64=0 --build-arg OTBR_OPTIONS='-DOT_DUA=ON -DOT_MLR=ON'"
```

Comment:
Build `latest` with 1.2 relative feature flags by default now?

Reply:
Note That the latest tag does not use any 1.2 feature. 
Btw, I changed the purpose of this PR. Labeled as wip.